python/ray/train/lightning/lightning_checkpoint.py

```diff
@@ -0,0 +1,46 @@
+import os
+import logging
+import pytorch_lightning as pl
+
+from inspect import isclass
+from typing import TYPE_CHECKING, Any, Dict, Optional, Type
+
+from ray.air.checkpoint import Checkpoint
+from ray.util.annotations import PublicAPI
+
+logger = logging.getLogger(__name__)
+
+
+@PublicAPI(stability="beta")
+class LightningCheckpoint(Checkpoint):
```

Comment:
can we use this as the API for LightningCheckpoint? https://gist.github.com/amogkam/6b3828cc8f260e4b58a186a92ef367d7

the checkpoint is just a single file, so a `from_file` type API would seem better than `from_directory` (just like with TensorflowCheckpoint)